static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -23,7 +23,7 @@ import { getForcedParticipationsFromUrl } from 'common/modules/experiments/ab-ur
 import {
     concurrentTests,
     engagementBannerTests,
-    epicTests,
+    epicTests as syncEpicTests,
```

Comment:
the `syncEpicTests` are the hardcoded tests, `asyncEpicTests` are fetched from config